Update scene quality listening test

hagenw opened the pull request:
I added this to the public data base and uploaded it to zenodo and modified the documentation accordingly.

This depends again on #34

hagenw committed:
Update scene quality listening test

Commit:
Merge branch 'master' into scene_quality

fietew commented:
2014-04-01: Scene related sound quality: 
1. Imho, a description of the different colors in the figure is missing. Without some explanation about how the colors are related to the kind/strength of degradation, the figure does not make too much sense.
2. Just an uninformed question by myself: Is a preference test automatically a paired comparison?  The sentence _"The third row contains a 1 or 2 indicated which of the two presented conditions was preferred by the listeners"_ suggests, that is a paired comparison. However, in the description, you called it preference test. Maybe one could add a additional adjective to clarify that.

hagenw committed:
Enhance description of scene quality experiment

hagenw commented:
1. I added a description of the different degradations and the color codings.
2. You are right, it does not mean automatically a paired comparison test (our 5.0 test was also a preference test, where you had to pick 1 out of 9). I added it to the description.
